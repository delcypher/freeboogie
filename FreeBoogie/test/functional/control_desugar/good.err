test1.bpl:8:3: Command is unreachable. 
test1.bpl:8:3: Command is unreachable. 
test1.bpl:8:3: Command is unreachable. 
test1.bpl:8:3: Command is unreachable. 
test1.bpl:8:3: Command is unreachable. 
