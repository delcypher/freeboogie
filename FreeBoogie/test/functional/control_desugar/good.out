 OK: p at test0.bpl:4:11
 OK: p at test1.bpl:4:11
 OK: p at test2.bpl:4:11
 OK: p at test3.bpl:4:11
 OK: p at test4.bpl:5:11
